src/ui/MessageContent/index.tsx

```diff
@@ -269,7 +269,7 @@ export default function MessageContent({
               message={message as SendableMessageType}
               userId={userId}
               isByMe={isByMe}
-              isUnavailable={(replyType === 'THREAD' && (channel?.joinedAt ?? 0 * 1000) > (message.parentMessage?.createdAt ?? 0))}
+              isUnavailable={(channel?.messageOffsetTimestamp ?? 0) * 1000 > (message.parentMessage?.createdAt ?? 0)}
```

Comment:
- Remove `replyType === 'THREAD'` because the quoted message should show the unavailable message regardless of the reply type.
- Use `messageOffsetTimestamp` instead of `joinedAt` because joinedAt can't cover the every case